whois-api/src/main/java/net/ripe/db/whois/api/rdap/RdapObjectMapper.java

```diff
@@ -693,11 +693,11 @@ private static String getAndHandleMultipleAttributes(final RpslObject rpslObject
         return attributes.get(0).getCleanValue().toString();
     }
 
-    private static void setGeoFeed(final RpslObject rpslObject, final Ip ip) {
-        ip.getRdapConformance().add(GEO_FEED_V1.getValue());
+    private static void setGeoFeed(final RpslObject rpslObject, final Ip ip, final String requestUrl) {
+        ip.getRdapConformance().add(GEO_FEED_1.getValue());
 
         if(rpslObject.containsAttribute(GEOFEED)) {
-            ip.setGeofeedv1_geofeed(rpslObject.getValueForAttribute(GEOFEED).toString());
+            ip.getLinks().add(new Link(requestUrl, "geo", rpslObject.getValueForAttribute(GEOFEED).toString(), null, "application/geofeed+csv"));
```

Comment:
Create a static field for the content-type as it's used twice

Reply:
Done